openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/ClassificationRepository.java

```diff
@@ -112,6 +117,39 @@ public TagLabel map(ResultSet r, org.jdbi.v3.core.statement.StatementContext ctx
     }
   }
 
```

Comment:
@sonika-shah whenever tag rename or glossary term rename happens. We should do this bulk operation in asynchronous fashion not in-line.
Might as well take the quartz approach and put a job that can handle failures and retries

Reply:
yes @harshach , the reindexing done via `reindexAcrossIndices`  is happening in [async](https://github.com/open-metadata/OpenMetadata/blob/main/openmetadata-service/src/main/java/org/openmetadata/service/search/elasticsearch/ElasticSearchClient.java#L1529C1-L1530C1) fashion